conversions/time_conversions.py

```diff
@@ -0,0 +1,36 @@
+"""
+USAGE :
+This function can be used for the following purposes:
+1. Unit conversion
+2. Standardization
+3. Problem solving
+4. Educational purposes
+
+
+
+REFERENCES :
+
+https://en.wikipedia.org/wiki/Unit_of_time
+https://en.wikipedia.org/wiki/Time
+
+"""
+
+
+def convert_time(time, from_unit, to_unit):
+    units = {"s": 1, "min": 60, "h": 3600, "d": 86400}
```

Comment:
Since you used single character for "second", "hour" and "day", use `"m"` for "minute" to be consistent.